docs/source/TaskEngineWorkflows.md

```diff
@@ -348,6 +354,39 @@ This workflow allows you to create an MP4 from a VOD asset
 }
 ```
 
+## build_thumbnails
+
+This workflow allows the generation of a thumbnail sprite and vtt file, which can then be used for video timeline preview thumbnails.
+
+## build_thumbnails: Parameters
+
+| Parameter Name    | Required |  Description | Default |
+| ----------------- | -------- | ------------ | ------- |
+| workflow          |Yes| Specify 'build_thumbnails'.||
+| source_file       |Yes| file name of the asset to generate the sprite/vtt from.||
```

Comment:
Include a comment that indicates that an HLS stream (URL) can be used as the source file.

Reply:
I would say that the m3u8 stream should be the default and remove mention of file name, since the file would need to be accessible/download to the job directory first.